src/scalper.ts

```diff
@@ -272,6 +274,16 @@ export class Scalper {
         console.log(err.stack);
       }
       hedgeCount++;
+
+      // Use loadFills() as a backup to websocket
+      let filledSize = await fillSize(perpMarket, this.connection, deltaOrderId);
```

Comment:
can these be const?
generally good to make const when possible

Reply:
Each run thru of deltaHEdge() it gets assigned a new value, not within a single run. Making a const shouldn't interfere with that then correct?

Reply:
Should not interfere. As long as it does not change within its scope, can be const